client/landing/site.landing/coffee/core/utils.coffee

```diff
@@ -283,10 +285,10 @@ module.exports = utils = {
 
   createTeam: (callbacks = {}) ->
 
-    # this part should never run actually because the signup flow in
-    # TeamAppController takes care of this. But i'll leave this here just in
-    # case.
-    unless token = utils.getPayment()?.token
+    isDefaultEnv = kd.config.environment is 'default'
+    token = utils.getPayment()?.token
+
+    if not isDefaultEnv and not token
```

Comment:
`unless ... and/or ...` usually makes me stop and try to understand the expression, i thought this was more readable, wdyt?